pages/about/contribute.md

```diff
@@ -1,94 +1,105 @@
+1. **Severity:** What type of functionality is impacted? Is there a workaround?
+1. **Priority:** Does this align with our vision and roadmap goals?
 
-### Want to propose something else?
+**Note:** We prioritize issues that affect accessibility.
 
-If you’d like to contribute something else that doesn’t fall into any of the above, we’d still love to hear about it. Just [create an issue](https://github.com/uswds/uswds/issues/new/choose) or a [discussion](https://github.com/uswds/uswds/discussions), and we can talk about it.
+These considerations help us decide if and when we can work on the issue. If the issue is accepted, we scheduled them for an upcoming sprint (a two-week work period).
 
-## How we prioritize
-
-Once you’ve submitted a contribution, we'll triage it based on the following considerations:
-
-1. **Size**: Can we accomplish this in a sprint or will this take longer?
-1. **Severity**: What type of functionality is impacted? Is there a workaround?
-1. **Priority**: Does this align with our vision and roadmap goals?
-
-**Note**: We prioritize issues that affect accessibility.
-
-These considerations help us decide if and when we can work on the issue. If the issue is accepted, we scheduled them for an upcoming sprint (a 2-week work period).
-
-You can stay up to date on the status of your contributions through [GitHub email notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications) (external link) and the assigned labels on the issue.
+You can stay up to date on the status of your contributions through [GitHub email notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications) and the assigned labels on the issue.
 
 ## Code of Conduct
+USWDS is committed to building a safe, welcoming, harassment-free culture for everyone.
+USWDS is a part of the Technology Transformation Services (TTS) within the General Services Administration (GSA) and we expect everyone to follow the GSA TTS Code of Conduct.
 
-USWDS is committed to building a safe, welcoming, harassment-free culture for everyone. USWDS is a part of the Technology Transformation Services (TTS) within the General Services Administration (GSA) and we expect everyone to follow the GSA TTS Code of Conduct.
-
-We encourage you to read the [LICENSE](https://github.com/uswds/uswds/blob/develop/LICENSE.md) and its [README](https://github.com/uswds/uswds/blob/develop/README.md). If you want to read more about our open source policy or have questions, check out the [18F Open Source Policy GitHub repository](https://github.com/18f/open-source-policy) or send us an [email](mailto:uswds@gsa.gov).
+We encourage you to read the [LICENSE](https://github.com/uswds/uswds/blob/develop/LICENSE.md) and the [LICENSE’s README](https://github.com/uswds/uswds/blob/develop/README.md).
```

Comment:
@sarah-sch The README link here seems to just point to our readme. The link is the same as what is currently on the site. Is this where it should point?

Reply:
@amyleadem - Hm, I am not sure if there is another README specific to the license. We could point to the [license section of the USWDS README](https://github.com/uswds/uswds/blob/develop/README.md#licenses-and-attribution), but it is not very extensive and just points people back to the License. @annepetersen and @thisisdano - any thoughts? If we get in a time crunch I think it is ok to publish as is and figure this out after the MC.

Reply:
If there isn't a README specific to the license, we could just cut it.